.github/workflows/infracost.yml

```
@@ -20,33 +20,43 @@ jobs:
       INFRACOST_API_KEY: ${{ secrets.INFRACOST_API_KEY }}
 
     steps:
+      - name: Skip Infracost when API key is unavailable
+        if: ${{ secrets.INFRACOST_API_KEY == '' }}
```

actionlint (1.7.11)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)